src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue

```diff
@@ -935,6 +965,23 @@ export default {
 			}
 		},
 
+		async setLobbyPermission(value) {
+			// Bitwise add or subtract lobby permission for a participant
```

Comment:
Now you did it already, but the API has the option to add/remove single permissions, so you wouldn't need to do this math yourself 😅👍

Reply:
It could be better to pass a single option without that arithmetic :see_no_evil:

Reply:
Then use method=set and only set the permission you want to toggle:
https://nextcloud-talk.readthedocs.io/en/latest/participant/#set-permissions-for-an-attendee